### ISSUE TITLE
Validation tables sometimes fails

### ISSUE THREAD

**Issue by tjgalvin** [tjgalvin/flint#56]
It appears that in some conditions the valibration tables task may fail. This was raised when processing SBID 39407

```
Encountered exception during execution:
Traceback (most recent call last):
  File "/scratch3/gal16b/mambaforge/envs/flint_noflagging/lib/python3.8/site-packages/prefect/engine.py", line 2099, in orchestrate_task_run
    result = await call.aresult()
  File "/scratch3/gal16b/mambaforge/envs/flint_noflagging/lib/python3.8/site-packages/prefect/_internal/concurrency/calls.py", line 293, in aresult
    return await asyncio.wrap_future(self.future)
  File "/scratch3/gal16b/mambaforge/envs/flint_noflagging/lib/python3.8/site-packages/prefect/_internal/concurrency/calls.py", line 318, in _run_sync
    result = self.fn(*self.args, **self.kwargs)
  File "/scratch3/gal16b/packages/flint_noflagging/flint/flint/prefect/common/imaging.py", line 625, in task_create_validation_tables
    validation_tables = create_validation_tables(
  File "/scratch3/gal16b/packages/flint_noflagging/flint/flint/validation.py", line 1232, in create_validation_tables
    xmatch_table, xmatch_file = make_xmatch_table(
  File "/scratch3/gal16b/packages/flint_noflagging/flint/flint/validation.py", line 925, in make_xmatch_table
    xmatch_table = Table(
  File "/scratch3/gal16b/mambaforge/envs/flint_noflagging/lib/python3.8/site-packages/astropy/table/table.py", line 875, in __init__
    init_func(data, names, dtype, n_cols, copy)
  File "/scratch3/gal16b/mambaforge/envs/flint_noflagging/lib/python3.8/site-packages/astropy/table/table.py", line 1425, in _init_from_dict
    self._init_from_list(data_list, names, dtype, n_cols, copy)
  File "/scratch3/gal16b/mambaforge/envs/flint_noflagging/lib/python3.8/site-packages/astropy/table/table.py", line 1244, in _init_from_list
    self._init_from_cols(cols)
  File "/scratch3/gal16b/mambaforge/envs/flint_noflagging/lib/python3.8/site-packages/astropy/table/table.py", line 1469, in _init_from_cols
    lengths = {len(col) for col in cols}
  File "/scratch3/gal16b/mambaforge/envs/flint_noflagging/lib/python3.8/site-packages/astropy/table/table.py", line 1469, in <setcomp>
    lengths = {len(col) for col in cols}
TypeError: len() of unsized object
```

My guess without looking is that there are no matches, or an empty table. More investigation needed.

**Comment by tjgalvin** [comment]
This has been fixed with #59
